roles/edpm_nova/molecule/default/prepare.yml

```diff
@@ -81,3 +81,11 @@
         owner: root
         group: root
         mode: 0755
+    - name: Create ssh_known_hosts file
+      become: true
+      ansible.builtin.file:
+        path: '/etc/ssh/ssh_known_hosts'
+        state: touch
+        owner: root
+        group: root
+        mode: 0655
```

Comment:
please, quote the octal number [1]

[1] https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-mode